actions/actions.py

```diff
@@ -15,23 +15,43 @@
 from rasa_sdk.forms import Action, FormAction
 
 
-class HealthCheckForm(FormAction):
+class HealthCheckProfileForm(FormAction):
     """HealthCheck form action"""
 
     def name(self) -> Text:
         """Unique identifier of the form"""
 
-        return "healthcheck_form"
+        return "healthcheck_profile_form"
 
     @staticmethod
     def required_slots(tracker: Tracker) -> List[Text]:
         """A list of required slots that the form has to fill"""
-        slots = ["province", "age", "cough", "exposure", "tracing"]
+        slots = [
+            "age",
+            "gender",
+            "province",
+            "medical_condition",
+        ]
+        slots_with_conditions = [
+            "age",
+            "gender",
+            "province",
+            "medical_condition",
+            "medical_condition_obesity",
+            "medical_condition_diabetes",
+            "medical_condition_hypertension",
+            "medical_condition_cardio",
+        ]
```

Comment:
Should we maybe have a `slots` and `conditions` variables, and do a `slots + conditions` when we want both? So that we don't have to remember to update in two places